lib/diagnostic.js

```diff
@@ -41,7 +81,11 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
-
+db.houses.deleteOne(
```

Comment:
This will delete the entire document rather than just the `motto` key from a particular document.  Use `update()` with `$unset` to remove keys.

```javascript
db.houses.update({
  name: 'House Arryn'
}, {
  $unset: {
    motto: ''
  }
});
```